src/server.cpp

```diff
@@ -132,15 +132,15 @@ void ServicePort::onAccept(Connection_ptr connection, const boost::system::error
 	}
 }
 
-Protocol_ptr ServicePort::make_protocol(bool checksummed, NetworkMessage& msg, const Connection_ptr& connection) const
+Protocol_ptr ServicePort::make_protocol(NetworkMessage& msg, const Connection_ptr& connection) const
 {
 	uint8_t protocolID = msg.getByte();
 	for (auto& service : services) {
 		if (protocolID != service->get_protocol_identifier()) {
 			continue;
 		}
 
-		if ((checksummed && service->is_checksummed()) || !service->is_checksummed()) {
+		if ((service->is_checksummed()) || !service->is_checksummed()) {
```

Comment:
isn't this always true?

Reply:
yeah, missed that when removing variable and its references